index.html

```diff
@@ -972,64 +972,127 @@ <h2>UTF-8 Character Encoding Form</h2>
     </aside>
 
 
+
+
+
+
 <section id="char_def" class="subtopic">
 <h3>Choosing a definition of 'character'</h3>
 
 
+<p>The term character is used differently in a variety of contexts and often leads to confusion when used outside of these contexts. In the context of the digital representations of text, a character can be defined as a small logical unit of text. Text is then defined as sequences of characters. While such an informal definition is sufficient to create or capture a common understanding in many cases, it is also sufficiently open to create misunderstandings as soon as details start to matter. In order to write effective specifications, protocol implementations, and software for end users, it is very important to understand that these misunderstandings can occur.</p>
```

Comment:
This is somewhat repetitious of the text just above in section 4's intro. That might be hard to avoid, given that the requirement "up there" is pretty similar to the first requirement in this section.

Reply:
I should have said more here.

Explanatory text should always occur below the mustard, so that someone who follows the link https://w3c.github.io/bp-i18n-specdev/index.html#char_define will see the needed supporting text.  They shouldn't have to scroll the page to be able to see the text higher up in order to understand things.

So the intro text is just that, and it's _that_ text that is repetitious, if anything is.  However, it thought it was a useful introduction for those who are reading specdev as a document, rather than trying to get clarity about a particular piece of mustard.